src/Fixer/DoctrineAnnotation/DoctrineAnnotationIndentationFixer.php

```diff
@@ -24,6 +24,11 @@
 final class DoctrineAnnotationIndentationFixer extends AbstractDoctrineAnnotationFixer
 {
     /**
+     * @var array
```

Comment:
not true until `fixAnnotations` is called.
I would remove the property and pass it directly as method parameter into `indentationCanBeFixed`